app/viewmodels/checkAnswers/ContactPhoneSummary.scala

```diff
@@ -22,20 +22,20 @@ import pages.ContactPhonePage
 import play.api.i18n.Messages
 import play.twirl.api.HtmlFormat
 import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
+import viewmodels.checkAnswers.Util.changeAction
 import viewmodels.govuk.summarylist._
 import viewmodels.implicits._
 
 object ContactPhoneSummary {
 
   def row(answers: UserAnswers)(implicit messages: Messages): Option[SummaryListRow] =
-    answers.get(ContactPhonePage).map {
+    answers.get(ContactPhonePage).orElse(Option("Not provided")).map {
```

Comment:
Very minor: Could be Some("Not provided") to avoid the Null check

Reply:
I generally follow the guide below from https://github.com/Originate/guide/blob/master/scala/BestPractices.md:

> By the principle of least astonishment, use Option.apply even if you "know" your reference can never be null. By being consistent, we avoid wondering whether a Some.apply in the code was intentional or something the developer overlooked:

This scenario is never going to throw a null pointer exception so I'm happy to use `Some("Not provided")`

Reply:
true 👍